plugins/vestige/vestige.cpp

```diff
@@ -338,14 +338,14 @@ void vestigeInstrument::loadFile( const QString & _file )
 
 	// if the same is loaded don't load again (for preview)
 	if (instrumentTrack() != NULL && instrumentTrack()->isPreviewMode() &&
-			m_pluginDLL == SampleBuffer::tryToMakeRelative( _file ))
+			m_pluginDLL == PathUtil::toShortestRelative( _file ))
 		return;
 
 	if ( m_plugin != NULL )
 	{
 		closePlugin();
 	}
-	m_pluginDLL = SampleBuffer::tryToMakeRelative( _file );
+	m_pluginDLL = PathUtil::toAbsolute( _file );
```

Comment:
This replacement looks a bit weird...
Will investigate later.

Reply:
Ok, this actually makes sense. A few lines later, m_pluginDLL gets passed into the constructor `new VstInstrumentPlugin(m_plugin DLL)` which inherits from VstPlugin's constructor. That constructor has logic to handle relative paths, but it was written for the old format of relative paths and wouldn't work with PathUtil's relative path. Sending an absolute path instead avoided this.

In the commits I just pushed, I changed this line, so now m_pluginDLL *is* made relative. Then I updated the constructor in VstPlugin to use toAbsolute instead of the old custom logic, so things should still work. Will test soon.

Reply:
This change breaks loading of FX. I think making oldRelativeUpgrade consider the UserVST base should fix this.